nevergrad/functions/control/core.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import numpy as np
+import typing as tp
+from nevergrad.parametrization import parameter as p
+from ..base import ExperimentFunction
+from .mujoco import GenericMujocoEnv
+
+class BaseFunction(ExperimentFunction):
+    def __init__(self, num_rollouts: int, random_state: tp.Optional[int] = None) -> None:
+        super().__init__(self._simulate, p.Array(shape=self.policy_dim))
+        self.num_rollouts = num_rollouts
+        self.random_state = random_state
+        self.register_initialization(num_rollouts=num_rollouts, random_state=random_state)
+        self.update_descriptors(num_rollouts=num_rollouts)
```

Comment:
```suggestion
        self.add_descriptors(num_rollouts=num_rollouts)
```
Sorry, got this one wrong :s